jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -758,6 +758,43 @@ ecma_op_is_compatible_property_descriptor (const ecma_property_descriptor_t *des
 
   return true;
 } /* ecma_op_is_compatible_property_descriptor */
+
+/**
+  * CompletePropertyDescriptor method for proxy internal method
+  *
+  * See also:
+  *          ECMA-262 v6, 6.2.4.5
+  */
+ void
+ ecma_op_to_complete_property_descriptor (ecma_property_descriptor_t *desc_p)
+ {
+  /* 4. */
+  if (!(desc_p->flags & (ECMA_PROP_IS_GET_DEFINED | ECMA_PROP_IS_SET_DEFINED)))
+  {
+    /* a. */
+    if (!(desc_p->flags & ECMA_PROP_IS_VALUE_DEFINED))
+    {
+      JERRY_ASSERT (desc_p->value == ECMA_VALUE_UNDEFINED);
+      desc_p->flags |= ECMA_PROP_IS_VALUE_DEFINED;
```

Comment:
The whole if can be removed only

`desc_p->flags |= ECMA_PROP_IS_VALUE_DEFINED` is enough, since if the flag is not present it will be applied, otherwise if it is present still remains present.